src/App.js

```diff
@@ -3,26 +3,52 @@ import Movies from './Movies';
 import Movie from './Movie';
 import Nav from './Nav';
 import movieData from './data';
+import fetchCalls from './apiCalls'
 import './App.css';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
+      movies:[],
+      singleMovieId:'',
       singleMovie:'',
+      error: '',
     }
   }
 
+componentDidMount = () => {
+  console.log('MOUNT')
+  fetchCalls.fetchData('movies')
+    .then(data => {
+      this.setState({movies: data.movies})
+    })
+    if (this.state.singleMovieId && !this.state.singleMovie) {
```

Comment:
This part of the code sets of a couple of alarm bells in my head because you might run into timing issues.  The code on 26 might run before you expect it to. If you want to run code right after a `setState` is _complete_ , then there is an optional callback function you can supply to `setState` to accomplish that (if running code right after a `setState` is _complete_ is your objective).